test/main.test.ts

```diff
@@ -0,0 +1,117 @@
+import { expect } from "chai";
+import * as ethers from "ethers";
+import { utils, Wallet, Provider, Contract, EIP712Signer, types } from "zksync-web3";
+import * as hre from "hardhat";
+import { Deployer } from "@matterlabs/hardhat-zksync-deploy";
+import { ZkSyncArtifact } from "@matterlabs/hardhat-zksync-deploy/dist/types";
+import { logCyan, sendFunds } from "./utils";
+
+const RICH_WALLET_PK = "0x3eb15da85647edd9a1159a4a13b9e7c56877c4eb33f614546d4db06a51868b1c";
```

Comment:
Could add `0x3eb15da85647edd9a1159a4a13b9e7c56877c4eb33f614546d4db06a51868b1c` to github secrets and reference in the CI and then use 👇 . More of a suggestion then anything. 

```suggestion
// load env file
import dotenv from "dotenv";
dotenv.config();

// load wallet private key from env file
const RICH_WALLET_PK = process.env.WALLET_PRIVATE_KEY || "";
```